### DISCUSSION TITLE
Location of read_media.html

### DISCUSSION THREAD

**Discussion by PhyDog** [bluenviron/mediamtx#3305] [Questions]
### Question

I'm trying to modify the page served with WebRTC but can't locate it on the latest Ubuntu server (headless).  All I want to do is make the controls=false and muted=false, the rest unchanged, but I can't locate read_media.html which should solve my issue.

**Comment by aler9** [comment]
Hello, in order to change those parameters, you can use query parameters:

```
http://localhost:8889/mystream?controls=0&muted=0
```

**Comment by PhyDog** [comment]
Thanks... I need to add some java and css to it but if it can't be 
modified in your server I will do it in nginx and serve that link 
inside.  It would have been easier to modify the page that mediamtx 
serves instead.

Mediamtx is rocking for me... thanks for your reply and  a great app!

[a comment by an automated account is not shown]
